src/resmom/catch_child.c

```diff
@@ -1098,14 +1099,17 @@ update_ajob_status_using_cmd(job *pjob, int cmd, int use_rtn_list_ext)
 	 */
 	if ((cmd != IS_RESCUSED_FROM_HOOK) ||
 		((pjob->ji_qs.ji_svrflags & JOB_SVFLG_HERE) != 0)) {
+		pjob->ji_last_resc_updated = time(NULL);
 		/* now append resources used */
 
 		encode_used(pjob, &rused.ru_attr);
 	}
 
 	/* now send info to server via rpp */
 
-	send_resc_used(cmd, 1, &rused);
+	if (!(pjob->ji_last_resc_updated < time_now - min_check_poll))	{
```

Comment:
This might be more readable as follows:
```suggestion
	if (pjob->ji_last_resc_updated >= (time_now - min_check_poll)) {
```

Reply:
Done